tutorcelery/patches/k8s-deployments

```diff
@@ -52,3 +52,31 @@ spec:
             name: openedx-config
 {% endfor %}
 {% endfor %}
+
+{% if CELERY_FLOWER -%}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: flower
+  labels:
+    app.kubernetes.io/name: flower
+spec:
+  selector:
+    matchLabels:
+      app.kubernetes.io/name: flower
+  template:
+    metadata:
+      labels:
+        app.kubernetes.io/name: flower
+    spec:
+      containers:
+        - name: flower-edxapp
+          image: docker.io/mher/flower:0.9.5
```

Comment:
There are new versions of the Flower [image](https://hub.docker.com/r/mher/flower/tags) worth testing. We can make this image configurable leaving `docker.io/mher/flower:0.9.5` as the default. Then we can bump it if we find the new versions are better.